src/main/java/ca/corefacility/bioinformatics/irida/ria/web/users/UsersAjaxController.java

```diff
@@ -46,6 +43,28 @@ public TableResponse<UserDetailsModel> getUsersPagedList(@RequestBody AdminUsers
 		return UIUsersService.getUsersPagedList(request);
 	}
 
+	/**
+	 * Create a new user
+	 *
+	 * @param userCreateRequest a {@link UserCreateRequest} containing details about a specific user
+	 * @param principal         a reference to the logged in user
+	 * @param locale            the logged in user's request locale
+	 * @return A list or errors
+	 */
+	@RequestMapping(value = "/create", method = RequestMethod.POST)
+	public ResponseEntity<Map<String, String>> createUser(@RequestBody UserCreateRequest userCreateRequest,
```

Comment:
You should be returning an AjaxResponse here.  There is an AjaxFormError response if you want to return the map

Reply:
Please see 73fe1adebcc57102ead361b043a357d26b648479.